test/cli/detailed-errors/test.out

```diff
@@ -0,0 +1,168 @@
+shapes.rb:10: Expected `{a: Integer, b: String}` but found `{}` for argument `x` https://srb.help/7002
```

Comment:
Is it expected that the first few of these don't have any explanations?

That's fine, we don't have to fix every error message in this PR, I just wanted to confirm.

Reply:
I don't quite understand this, I don't think variance is involved here. Maybe we can walk through this together tomorrow.